test/test_frame_2d.py

```diff
@@ -5,18 +5,26 @@
 def test_image_context_manager(physical_camera_frame_2d):
     import zivid
 
-    with physical_camera_frame_2d.image_rgba() as image:
-        assert image is not None
-        assert isinstance(image, zivid.Image)
+    with physical_camera_frame_2d.image_rgba() as image_rgba:
+        assert image_rgba is not None
+        assert isinstance(image_rgba, zivid.Image)
+
+    with physical_camera_frame_2d.image_bgra() as image_bgra:
+        assert image_bgra is not None
+        assert isinstance(image_bgra, zivid.Image)
 
 
 @pytest.mark.physical_camera
 def test_image(physical_camera_frame_2d):
     import zivid
 
-    image = physical_camera_frame_2d.image_rgba()
-    assert image is not None
-    assert isinstance(image, zivid.Image)
+    image_rgba = physical_camera_frame_2d.image_rgba()
+    assert image_rgba is not None
+    assert isinstance(image_rgba, zivid.Image)
+
+    image_bgra = physical_camera_frame_2d.image_bgra()
+    assert image_bgra is not None
+    assert isinstance(image_bgra, zivid.Image)
```

Comment:
Can you also add some sort of test that checks that the RGBA and BGRA from the same Frame2D contain the same image?

Something like (pseudocode):
```
rgba.r == bgra.r
rgba.g == bgra.g
rgba.b == bgra.b
rgba.a == bgra.a
```

Reply:
Same for the frame.point_cloud data when you add that.